package.json

```diff
@@ -11,7 +11,7 @@
     ]
   },
   "engines": {
-    "atom": "^1.0.8"
+    "atom": "*"
```

Comment:
This should be `">=1.0.8 <2.0.0"`, `*` would allow _any_ version which wouldn't exactly be true...